manifests/init.pp

```diff
@@ -69,10 +69,16 @@
 #   Set to true if corosync_votequorum should be used as quorum provider.
 #   Defaults to false.
 #
+# [*expected_votes*]
+#   Number of votes required to make a quorum.
+#   Defaults to undef.
+#
 # [*quorum_members*]
-#   Array of quorum member hostname. This is required if set_votequorum
-#   is set to true.
-#   Defaults to undef,
+#   Array of quorum member hostname. 
+#   Defaults to undef.
+#
+# [*two_node*]
+#   Set to 1 to enable special quorum behaviour when only using two nodes.
```

Comment:
`corosync` configuration makes me cry.

We should support `1` here as it is what corosync uses internally, but I believe that we *also* should accept `true/false` and munge them into `1/0`.